### PR TITLE
Add customisation for end of course label

### LINKED ISSUE #1: Allow the block title to be customizable
https://github.com/IONISx/xblock-feedback/blob/master/feedback/static/html/view-feedback.html#L10

Just like the button label is.

### PR DESCRIPTION
**Pull request by fcheval**
Fix #1

### CODE REVIEW

File: feedback/defaults.py [hunk 1]
```diff
@@ -6,8 +6,9 @@
 
 DISPLAY_NAME = "Feedback"
 
-POST_URL = '#'
+POST_URL = "#"
 
-EXIT_LABEL = 'Save and exit'
+EXIT_LABEL = "Save and exit"
+END_COURSE_LABEL = "Congratulation, you finished this course !"
```
Comment (tusbar) [inline feedback/defaults.py]: No space before the exclamation mark in English.

File: feedback/feedbackblock.py [hunk 2]
```diff
@@ -56,6 +56,12 @@ class FeedbackXBlock(XBlock):
         help="Label for button exit.",
     )
 
+    end_course_label = String(
```
Comment (tusbar) [inline feedback/feedbackblock.py]: it’s not a label, it’s a title

and why not call it just title?

**Comment by tusbar** [comment]
This is missing the default value from `XBLOCK_SETTINGS`.

File: feedback/feedbackblock.py (hunk 2 above)
Comment (tusbar) [inline feedback/feedbackblock.py]: and why the `end_course` prefix? ending the course is the purpose of this XBlock.

Comment (fcheval) [inline feedback/feedbackblock.py]: done

File: feedback/defaults.py (hunk 1 above)
Comment (fcheval) [inline feedback/defaults.py]: done

File: feedback/feedbackblock.py [hunk 3]
```diff
@@ -108,6 +115,10 @@ def student_view(self, context=None):
                 s = settings.get('exit_label')
                 exit_label = s if s else EXIT_LABEL
 
+            if title == "":
+                s = settings.get('title')
+                title = s if s else TITLE
+
```
Comment (tusbar) [inline feedback/feedbackblock.py:121]: This will not work, as the default value is `TITLE`.

Comment (fcheval) [inline feedback/feedbackblock.py:121]: ok, done

**Comment by tusbar** [comment]
Now squash all the commits into one – and :+1:.

**Pull request merged**